src/sst/core/testingframework/sst_unittest_support.py

```diff
@@ -778,6 +778,27 @@ def log_fatal(errstr):
     log_forced(finalstr)
     sys.exit(2)
 
+###
+
+def log_testing_note(note_str):
+    """ Log a 'FATAL:' message.
```

Comment:
This 'FATAL:' contradicts the "NOTE : " below.  Cut and paste leftover?

Reply:
Yup, copy/paste error.  Fixed.